Javascript/calculate.js

```diff
@@ -510,18 +510,14 @@ function calculateTalismanEffects() {
         }
 
     }
-    rune1Talisman = 0;
-    rune2Talisman = 0;
-    rune3Talisman = 0;
-    rune4Talisman = 0;
-    rune5Talisman = 0;
-    for (let i = 1; i <= 7; i++) {
-        rune1Talisman += window["talisman" + i + "Effect"][1]
-        rune2Talisman += window["talisman" + i + "Effect"][2]
-        rune3Talisman += window["talisman" + i + "Effect"][3]
-        rune4Talisman += window["talisman" + i + "Effect"][4]
-        rune5Talisman += window["talisman" + i + "Effect"][5]
-    }
+    const talismansEffects = [talisman1Effect, talisman2Effect, talisman3Effect, talisman4Effect, talisman5Effect, talisman6Effect, talisman7Effect];
+    const runesTalisman = [0, 0, 0, 0, 0, 0];
+    talismansEffects.forEach((talismanEffect) => {
+        talismanEffect.forEach((levels, runeNumber) => {
+            runesTalisman[runeNumber] += levels;
+        });
+     });
+    [, rune1Talisman, rune2Talisman, rune3Talisman, rune4Talisman, rune5Talisman] = runesTalisman;
```

Comment:
```suggestion
    [rune1Talisman, rune2Talisman, rune3Talisman, rune4Talisman, rune5Talisman] = runesTalisman;
```
Is this a typo ? Or only something I did not know one could do

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment